lib/record_store/record/sshfp.rb

```diff
@@ -0,0 +1,56 @@
+module RecordStore
+  class Record::SSHFP < Record
+    attr_accessor :algorithm, :fptype, :fingerprint
+
+    # https://www.iana.org/assignments/dns-sshfp-rr-parameters/dns-sshfp-rr-parameters.xhtml
+    module Algorithms
+      RESERVED = 0
+      RSA = 1
+      DSA = 2
+      ECDSA = 3
+      ED25519 = 4
+      UNASSIGNED = 5
+      ED448 = 6
+    end
+
+    module FingerprintTypes
+      RESERVED = 0
+      SHA_1 = 1
+      SHA_256 = 2
+    end
+
+    FINGERPRINT_REGEX = /\A[[:xdigit:]]+\Z/
```

Comment:
You probably want to match against the entirety of the string.

```suggestion
    FINGERPRINT_REGEX = /\A[[:xdigit:]]+\z/
```